### PR TITLE
🎉  Thumbor

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Open tasks:

- [ ] Check if thumbor is running locally, only then register middleware
- [ ] Remove images from amp.dev image's static dest to not duplicate them
- [ ] Marry with #3978
- [ ] Add to Travis job
- [ ] Initial setup for prod on GCloud

### CODE REVIEW

File: platform/lib/routers/thumbor.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  return join('/static/', '/**/', `*.${extension}`);
+});
+
+thumborRouter.get(imagePaths, (request, response, next) => {
+  // Thumbor expects SECURITY_KEY as URL partial
+  request.url = join(SECURITY_KEY, request.url);
+
+  proxy.web(request, response, proxyOptions, (error) => {
+    log.info('Proxy error', error);
+    response.status(502).end();
```
Comment (sebastianbenz) [inline platform/lib/routers/thumbor.js:47]: should we fallback to the local file instead? My idea was to encode the requested width via query param which would make it easy to fallback to the local filesystem.

Comment (matthiasrohmer) [reply platform/lib/routers/thumbor.js:63]: Exactly how it's done now 😉

**Comment by matthiasrohmer** [comment]
Merging to see how it all ends up on staging. PR for production to follow.

**Comment by sebastianbenz** [comment]
Nice!

**Pull request merged**